packages/sw-lib/src/lib/sw-lib.js

```diff
@@ -351,6 +352,25 @@ class SWLib {
       requestWrapper: new RequestWrapper(wrapperOptions),
     });
   }
+
+  /**
+   * This method will register any default routes the library will need.
+   */
+  _registerDefaultRoutes() {
+    const revisionedManager = this._precacheManager.getRevisionedCacheManager();
+    const cacheFirstHandler = this.cacheFirst({
+      cacheName: revisionedManager.getCacheName(),
+    });
+
+    const route = new this.Route({
+      match: ({url, event}) => {
+        const cachedUrls = revisionedManager.getCachedUrls();
```

Comment:
Calling `getCachedUrls()` [is non-trivial](https://github.com/GoogleChrome/sw-helpers/blob/a1451310054b75e14882703a4cdb06dd944fdcca/packages/sw-precaching/src/lib/controllers/base-cache-manager.js#L53). How about you initialize `cachedUrls` outside of the `match()` function and then reuse the value each time `match()` is called?